browser/ephemeral_storage/ephemeral_storage_browsertest.cc

```diff
@@ -165,17 +165,32 @@ void HttpRequestMonitor::Clear() {
 
 EphemeralStorageBrowserTest::EphemeralStorageBrowserTest()
     : https_server_(net::EmbeddedTestServer::TYPE_HTTPS) {
```

Comment:
= default

Reply:
this is actually not possible due to `https_server_(...)`